app/templates/content/index.hbs

```diff
@@ -90,17 +90,23 @@
                         </button>
                     </div>
 
-                    {{#if model.links.preprint_doi}}
-                        <div class="p-t-xs">
-                            <h4 class="p-v-md f-w-md"><strong>{{t "content.preprint_doi"}}</strong></h4>
+                    <div class="p-t-xs">
+                        <h4 class="p-v-md f-w-md"><strong>{{t "content.preprint_doi"}}</strong></h4>
+                        {{#if model.links.preprint_doi}}
```

Comment:
Would you mind updating this section to use `model.preprintDoiUrl` instead of `model.links.preprint_doi`?

...and if you're not opposed to gentle scope creep, updating the next section to use `model.articleDoiUrl` instead of `model.doi` and `doiUrl`?

Reply:
👍